index.bs

```diff
@@ -475,7 +475,8 @@ When this method is invoked, the user agent MUST execute the following algorithm
     1. If |extension| is not supported by this client platform, then [=continue=].
 
     1. Otherwise, let |result| be the result of running |extension|'s [=client processing=] algorithm on |argument|. If the
-        algorithm returned an error, [=continue=].
+        algorithm returned an error, return [=a promise rejected with=] a {{DOMException}} whose name is
```

Comment:
I think you want just "... rejected with=] a {{[TypeError](https://heycam.github.io/webidl/#exceptiondef-typeerror)}}, ..."